rcl/src/rcl/wait.c

```diff
@@ -604,20 +673,25 @@ rcl_wait(rcl_wait_set_t * wait_set, int64_t timeout)
   // Items that are not ready will have been set to NULL by rmw_wait.
   // We now update our handles accordingly.
 
+  bool timer_is_ready = false;
```

Comment:
can we rename this to `any_timer_is_ready` ?
I was confused to see both this variable and the `bool is_ready` at line 686 (which could be renamed into `bool this_timer_is_ready`, but it's secondary)